media/css/pebbles/includes/mixins/_typography.scss

```diff
@@ -0,0 +1,81 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+// Font families
+@mixin fira-sans {
+    font-family: $font-fira-sans;
+}
+
+@mixin fira-sans-light {
+    font-family: 'Fira Sans Light', X-LocaleSpecific-Light, $font-fira-sans;
```

Comment:
This may not be something we need to address here, but now we're using Fira do we still need all the extra `X-LocaleSpecific` logic in bedrock? It was added to help overcome some shortcomings in Open Sans (iirc), but Fira now has much better support for different character sets.